core/modules/syncer.js

```diff
@@ -134,6 +135,14 @@ function Syncer(options) {
 			$tw.utils.copyToClipboard($tw.utils.getSystemInfo() + "\n\nLog:\n" + self.logger.getBuffer());
 		});
 	}
+	// Handle loading of skinny tiddlers on demand.
+	$tw.hooks.addHook("th-navigating", (evt) => {
```

Comment:
TW core code still uses ES5 syntax. 

 - So arrow functions aren't allowed yet.
 -  core functions use `event` as the variable name for events. 
 - `const` and `let` needs to be `var` because of ES5

Reply:
Oops. Updated.